### PR TITLE
Issue #122

### PR DESCRIPTION
**Pull request by keven321**
Added two input fields for first and last names to the waitlist form. They don't have any functionality though.

### CODE REVIEW

File: components/ui/molecules/JoinWaitlist.tsx [hunk 1]
```diff
@@ -49,13 +49,35 @@ export function WaitlistForm({ accent, isLight }: WaitlistFormProps) {
 
   return (
     <form onSubmit={handleJoinWaitlist}>
-      <div className={`w-full flex border border-[${accent}] rounded-md p-1`}>
+      <div
+        className={`w-full flex flex-wrap border border-[${accent}] rounded-md p-1`}
+      >
+        <div className="flex flex-row">
```
Comment (CeoFred) [inline components/ui/molecules/JoinWaitlist.tsx:55]: try this className instead "columns-2 gap-2" remove the flex and flex-row

**Comment by CeoFred** [comment]
view this https://tailwindcss.com/docs/columns, should help.

**Comment by keven321** [comment]
I added the changes let me know if you need me to change anything else.

**Comment by CeoFred** [comment]
I'll be closing this as the solution is implemented at #130

**Pull request closed without merging**